etc/fastly_edge_modules/netacea_integration.json

```diff
@@ -3,44 +3,44 @@
e can no longer trust them:\n        unset req.http.X-Netacea-UserId;\n        unset req.http.netacea_match;\n        unset req.http.netacea_mitigate;\n        unset req.http.netacea_captcha;\n      } else {\n        if (time.is_after(now, std.time(var.netacea_cookie_expiry, now)) || digest.base64(var.netacea_real_client_ip_time_hash) != var.netacea_client_ip_time_hash ) {\n          # Cookie is expired or IP has changed\n          set req.http.netacea_mitata_must_reauthenticate = \"1\";\n        }\n      }\n    } else {\n      # Delete cookie which does not match regex\n      unset req.http.Cookie:_mitata;\n    }\n  }\n\n  # Cannot have _mitatacaptcha without _mitata\n  if (!req.http.Cookie:_mitata) {\n    unset req.http.Cookie:_mitatacaptcha;\n  }\n}\n\n# Read _mitatacaptcha cookie to assess whether the user has passed captcha\n# Unsets _mitatacaptcha if signature is not valid\n# Uses:\n#   req.http.Cookie:_mitatacaptcha\n#   req.http.X-Netacea-UserId\n# May set:\n#   req.http.X-Netacea-Captcha-Status: either \"captcha_pass\" or \"captcha_fail\"\nsub process_netacea_mitatacaptcha_cookie {\n  declare local var.netacea_mitSvc_secret STRING;\n  set var.netacea_mitSvc_secret = table.lookup(Netacea_Config, \"secret_key\");\n\n  declare local var.netacea_cookie_sig STRING;\n  declare local var.netacea_cookie_payload STRING;\n  declare local var.netacea_cookie_HMAC STRING;\n  declare local var.netacea_cookie_expiry STRING;\n  declare local var.netacea_cookie_captcha_status STRING;\n\n  if (req.http.Cookie:_mitatacaptcha) {\n    if (req.http.Cookie:_mitatacaptcha ~ \"^(.*)_\\/@#\\/(\\d)_\\/@#\\/(\\d+)$\") {\n      set var.netacea_cookie_sig = re.group.1;\n      set var.netacea_cookie_captcha_status = re.group.2;\n      set var.netacea_cookie_expiry = re.group.3;\n\n      # Include User ID in payload for hash calculation.\n      set var.netacea_cookie_payload = var.netacea_cookie_captcha_status + \"_/@#/\" + var.netacea_cookie_expiry + \"_/@#/\" + req.http.X-Netacea-UserId;\n\n      # Calculate HMAC\n      set var.netacea_cookie_HMAC = digest.hmac_sha256(var.netacea_mitSvc_secret, var.netacea_cookie_payload);\n      if (var.netacea_cookie_HMAC ~ \"0x(.*)\") {\n        set var.netacea_cookie_HMAC = re.group.1;\n      }\n\n      # Delete cookie with invalid HMAC\n      if (var.netacea_cookie_sig != digest.base64(var.netacea_cookie_HMAC)) {\n        unset req.http.Cookie:_mitatacaptcha;\n      }\n\n      # Delete expired cookie\n      if (time.is_after(now, std.time(var.netacea_cookie_expiry, now))) {\n        unset req.http.Cookie:_mitatacaptcha;\n      }\n\n      if (req.http.Cookie:_mitatacaptcha) {\n        if (var.netacea_cookie_captcha_status == \"1\") {\n          set req.http.X-Netacea-Captcha-Status = \"captcha_pass\";\n        } else {\n          set req.http.X-Netacea-Captcha-Status = \"captcha_fail\";\n        }\n      }\n    } else {\n      # Delete cookie which does not match regex\n      unset req.http.Cookie:_mitatacaptcha;\n    }\n  }\n}",
```

Comment:
You are sure you want to turn of TLS cert verification ?

Reply:
Hey, turns out we didn't, was a miscommunication internally. Resolved now.